### PR TITLE
Implemented tools library for s3bench and warp. Copy Object Workload

### PR DESCRIPTION
**Pull request by ketanarlulkar**
# Problem Statement
-  Implemented tools library for s3bench and warp
-  Copy Object Workload

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed
-  [ ] Attach test execution logs

# Review Checklist 
  Checklist for Author
-  [ ] JIRA number/GitHub Issue added to PR
-  [ ] PR is self reviewed
-  [ ] Jira and state/status is updated and JIRA is updated with PR link
-  [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe

### CODE REVIEW

**Review by priyankaborawake — APPROVED** [review]

File: config/s3/workloads/s3_copy_object.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+test_1:
```
Comment (DhobaleVishal) [inline config/s3/workloads/s3_copy_object.yaml]: Workload directory structure should be tool specific for ex s3api: config/s3/s3api/test.yaml

File: workload/s3/copy_object.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                        bucket=bucket1, key=object1, ranges=f'bytes={offset}-{range_read + offset}')
+                    checksum2 = await self.get_s3object_checksum(
+                        bucket=bucket2, key=object2, ranges=f'bytes={offset}-{range_read + offset}')
+                    assert checksum1 == checksum2, f"SHA256 of original range = {checksum1}\n" \
+                                                   f"SHA256 of copied range = {checksum2}"
+                else:
+                    # Download source and destination object and compare checksum
+                    checksum1 = await self.get_s3object_checksum(bucket1, object1)
+                    checksum2 = await self.get_s3object_checksum(bucket2, object2)
+                    assert checksum1 == checksum2, f"SHA256 of original object = {checksum1}\n" \
```
Comment (DhobaleVishal) [inline workload/s3/copy_object.py]: Assert statement should be clear.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

File: workload/s3/copy_object.py (hunk 2 above)
Comment (ketanarlulkar) [reply workload/s3/copy_object.py]: Done

File: config/s3/workloads/s3_copy_object.yaml (hunk 1 above)
Comment (ketanarlulkar) [reply config/s3/workloads/s3_copy_object.yaml]: Done

**Comment by VikasKumarC** [comment]
@ketanarlulkar  Please fix all codacy issues

**Comment by VikasKumarC** [comment]
@ketanarlulkar  resolve codacy issues

**Comment by VikasKumarC** [comment]
@ketanarlulkar  Resolve conflicts 
@DhobaleVishal  Please check

File: workload/s3/copy_object.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python
```
Comment (DhobaleVishal) [inline workload/s3/copy_object.py]: @ketanarlulkar, @VikasKumarC  Can we stick with one directory structure? pls suggest so we can follow same across corio.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

**Review by DhobaleVishal — APPROVED** [review]

**Pull request merged**